src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -114,7 +117,9 @@ Profile::~Profile()
 
 GUID Profile::GetGuid() const noexcept
 {
-    return _guid;
+    // This can throw if we never had our guid set to a legitimate value.
+    THROW_HR_IF_MSG(E_FAIL, !_guid.has_value(), "Profile._guid always expected to have a value");

```

Comment:
Is this a localization problem?

Reply:
Oh no that's just an internal exception. I don't think it's possible to actually hit at runtime, unless someone screws something up